### ISSUE TITLE
Settings button in top left is read as "unlabeled button" with Talkback

### ISSUE THREAD

**Issue by ellenpoe** [signalapp/Signal-Android#8728]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.

### Steps to reproduce
- Turn talkback on
- Open signal and tap on the button in the top left that contains your initials and leads to a settings page.
- Observe the verbalized output

**Actual result:** "Unlabeled Button" is verbalized
**Expected result:** "Settings shortcut, button" or "Settings shortcut" would be the behavior I'd suggest in this case, depending on system verbosity settings.

### Screenshots
This is super easy to repro and I the screenshot showing this behavior would contain personal information so please contact me if you need a screenshot.

### Device info

**Device:** Pixel 2
**Android version:** 9
**Signal version:** 4.36.2

### Link to debug log
This bug is just to track an android:contentDescription issue so I don't think you need a log (again, privacy concerns).
